src/compile-cache.js

```diff
@@ -0,0 +1,131 @@
+'use strict'
+
+var path = require('path')
+var CSON = require('season')
+var fs = require('fs-plus')
+var _ = require('underscore-plus')
```

Comment:
:+1: Good call. It seems like this would be addressed if we did the following:

1. Defer the require of `coffee-script` like we talked about
2. Move the setup for CSON back *out* of `compile-cache.js`, and into `index.js` after the `ModuleCache` is set up
3. Don't use `_.each` in `compile-cache`

That way, no new libraries will be required before the module cache. What do you think?